core-dev/packages/rocq-stdlib/rocq-stdlib.9.0.dev/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "The Rocq Proof Assistant -- Standard Library"
+description: """
+Rocq is a formal proof management system. It provides
+a formal language to write mathematical definitions, executable
+algorithms and theorems together with an environment for
+semi-interactive development of machine-checked proofs.
+
+Typical applications include the certification of properties of
+programming languages (e.g. the CompCert compiler certification
+project, or the Bedrock verified low-level programming library), the
+formalization of mathematics (e.g. the full formalization of the
+Feit-Thompson theorem or homotopy type theory) and teaching.
+
+This package includes the Rocq Standard Library, that is to say, the
+set of modules usually bound to the Stdlib.* namespace."""
+maintainer: ["The Rocq standard library development team"]
+authors: ["The Rocq development team, INRIA, CNRS, and contributors"]
+license: "LGPL-2.1-only"
+homepage: "https://coq.inria.fr/"
+doc: "https://coq.github.io/doc/"
+bug-reports: "https://github.com/coq/stdlib/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "rocq-runtime"
+  "rocq-core" {= version}
+  "odoc" {with-doc}
+]
+depopts: ["coq-native"]
+dev-repo: "git+https://github.com/coq/stdlib.git"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dev/with-rocq-wrap.sh"
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+
+url {
+  src: "git+https://github.com/coq/stdlib.git#V9.0+rc1"
```

Comment:
The URL does not match the package version.

Reply:
That's the closest thing we have

Reply:
Yes, that was my point, the version of the package should just be 9.0+rc1. There is no reason to make it look like it follows a branch.